packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -375,8 +372,9 @@ export const getCartDetails = (payload = {}) => {
 
             // check if the guest cart has expired
             if (response && response.status === 404) {
-                // if so, then delete the cached ID...
-                // in contrast to the save, make sure storage deletion is
+                // if so, then delete the cached ID from local storage.
+                // The reducer handles clearing out the bad ID from Redux.
```

Comment:
Nothing changed here, I just updated the comment to make this part clear because this function and `removeItemFromCart` differ from the others (they don't call `removeCart`).